ragna/assistants/_anthropic.py

```diff
@@ -10,10 +10,6 @@ class AnthropicAssistant(HttpApiAssistant):
     _STREAMING_PROTOCOL = HttpStreamingProtocol.SSE
     _MODEL: str
 
-    @classmethod
-    def _extra_requirements(cls) -> list[Requirement]:
-        return [PackageRequirement("httpx_sse")]
```

Comment:
@dillonroach Driveby. The requirements for the streaming are handled automatically.